### DISCUSSION TITLE
Show a popup that includes only the window's title (without a thumbnail)

### DISCUSSION THREAD

**Discussion by dorki** [lawand-dot-io/taskbar#107] [Feature requests]
Hi there!
great tool ! very helpfull.

would it be possible to show the window title even when the thumbnails feature is disabled ?
thanks !

**Comment by lawand-dot-io** [comment]
Hello. The window titles are already shown in the tasks and you can adjust the maximum task width, but I guess what you're talking about is in case there are many windows, the titles become short, so you need a way to see the full window title, for example on hover. Is that right?

**Comment by dorki** [comment]
Hi there thank u very much for the quick response.

I actually prefer showing only icons to begin with, and then my only option
is if there was a hover title.
But i guess u are correct that this will also be handy for those who likw
the title inlined

**Reply by lawand-dot-io** [reply]
there's already an option to hide the window titles

**Comment by dorki** [comment]
yes thats great!
but then since i dont have the hover thingy, i cant see the title :)

**Reply by lawand-dot-io** [reply]
ok, now I understand exactly what you mean. I will consider this. cheers

**Comment by dorki** [comment]
Thanks for your understanding.
